saleor/graphql/product/mutations/attributes.py

```diff
@@ -515,6 +555,130 @@ def perform_mutation(cls, _root, info, **data):
         return cls(product_type=product_type)
 
 
+class PageAttributeAssign(BaseMutation):
+    page_type = graphene.Field(PageType, description="The updated page type.")
+
+    class Arguments:
+        page_type_id = graphene.ID(
+            required=True,
+            description="ID of the page type to assign the attributes into.",
+        )
+        attribute_ids = graphene.List(
+            graphene.NonNull(graphene.ID),
+            required=True,
+            description="The IDs of the attributes to assign.",
+        )
+
+    class Meta:
+        description = "Assign attributes to a given page type."
+        error_type_class = ProductError
```

Comment:
This shouldn't use `ProductError`, instead rather `PageError`? I'm now thinking that maybe `AttributeError` would work for all mutations related to attributes but that's just a thought.

Reply:
IMO using `AttributeError` everywhere will be the best solution. But I'm not sure if we want to change the other mutations.

Reply:
After discussion: `AttributeError` will be introduced in separate PR.